src/envs/otoole_convert.yaml

```diff
@@ -0,0 +1,7 @@
+name: otoole_convert
+channels:
+  - conda-forge
+dependencies:
+ - pip
+ - pip: 
```

Comment:
Do you need both of these pip items or only one? I'm not sure.

Reply:
Yeah, good catch. I originally only had the second one, but snakemake/conda wans't happy with that